lib/proxy-client.js

```diff
@@ -95,7 +97,11 @@ ProxyClient.prototype.request = function request(method, path) {
   var start = Date.now();
   var href = self.getUrl(path);
 
-  return superagent(method, href)
+  var sa = superagent(method, href);
+  if (self.agent) {
+    sa.agent(self.agent);
```

Comment:
I originally understood ```Request.agent``` returned the agent on a falsy, but that's only with zero arguments. I'll change it.